qemu/tests/driver_in_use.py

```diff
@@ -66,6 +66,13 @@ def run_bg_test_simu(bg_stress_test):
                 utils_test.run_virt_sub_test, (test, params, env),
                 {"sub_type": bg_stress_test})
             stress_thread.start()
+
+        # set_bg_stress_flag is only used for balloon cases
+        if params.get("set_bg_stress_flag", "no") == "yes":
+            if not utils_misc.wait_for(lambda: env.get("memory_stable"),
+                                       240, 0, 1):
+                test.error("Guest memory status is not stable!")
```

Comment:
A generalized approach should be:
```python
for event in params.get("check_setup_events", "").strip().split():
    if not utils_misc.wait_for(lambda: env.get(event), ...):
        test.error("Background test not in ready state since haven't received event '%s'", event)
```
----
```
check_setup_events = memory_stable
```

Reply:
Modified,thanks